build/aliases/aliases.txt

```diff
@@ -0,0 +1,28 @@
+root	= pushd $Root\
+build	= $Root\build\build.cmd
+
+..	= cd ..
+n	= notepad.exe
+n++	= notepad++.exe
+
+ga	= git add
```

Comment:
I am not sure if we need to make it available in the repo. Not everyone will use aliases. It boils down to individual preferences. Since it is not core to developing Mobius, I recommend not adding it to repo